src/runtime/less/ts-mixins.less

```diff
@@ -255,40 +255,9 @@
 	flex-basis: @value;
 }
 
-/* Loading bar ................................................................. */
+// Loading bar .................................................................
 .ts-mixin-progress-bar-gradient(@color1: black, @color2: white) {
-	background-image: -webkit-gradient(
```

Comment:
What's happened here?

Reply:
we don't need browser prefixes for this feature

Reply:
Nice